src/components/feedback/feedback.html

```diff
@@ -40,8 +40,11 @@ <h3>You can tell us here how we can improve, or you can chat with us on <a href=
                         </tr>
                         <tr v-if="message.visible">
                             <td>
-                                <div v-if="message.from == 'FromUser'">
-                                    <pre style="background-color: #ffffff;">{{message.contents}}</pre>
+                                <div v-if="message.from == 'FromUser'" style="background-color: #ffffff;">
```

Comment:
A simplification would be to move the if statement into the style (I wish i did that when i originally wrote it).

Reply:
what is the syntax for that?